### DISCUSSION TITLE
AsyncAppender missing tid

### DISCUSSION THREAD

**Discussion by hogger0** [apache/skywalking#12220] [Q&A] [answered]
https://skywalking.apache.org/docs/skywalking-java/v9.2.0/en/setup/service-agent/java-agent/application-toolkit-logback-1.x/

logback-spring.xml According to the official documentation configuration, AsyncAppender cannot get tid, whether it is grpc or console all print N/A. Switching to a synchronous Appender is fine.
agen version: 9.2
Apm-tool-logback-1.x: 9.2
springboot: 3.2.5

**Comment by wu-sheng** [comment, marked as answer]
You don't need AsyncAppender for SkyWalking log collecting. There is async mechanism internally. Use sync appender.

**Reply by hogger0** [reply]
thx
